src/app/content/components/Topbar/styled.tsx

```diff
@@ -53,7 +53,7 @@ const closeIconStyles = css`
   color: #cdcdcd;
 `;
 
-const shadow = css`
+export const shadow = css`
```

Comment:
How is Shadow currently applied?

Reply:
SearchPrintWrapper and MobileSearchContainer use `shadow` for the `box-shadow` styles, and those wrappers really contain most of the styles that make the toolbar work, so this change dry-s it up and replaces my hard-coded `box-shadow`. Maybe we want this to be a styled wrapper with the single style rule instead?